plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -730,6 +730,18 @@ void generateFile(ref Database db, ref FileCtx ctx) @trusted {
 
     line.addChild("span", "1:").addClass("line_nr");
     auto mut_data = appender!(string[])();
+
+    
+
+
+    mut_data.put("var g_mut_description = {};");
+    mut_data.put(format!"g_mut_description[%s] = %s;"("''", "'Undefined'"));
+    foreach(kind; [EnumMembers!MutationKind]){
```

Comment:
instead of the condition in the loop i recommend using `filter`.

```d
foreach (kind; mutationDescription.byKeyValue.filter!(a => a.key != MutationKind.all)) {..}
```